inscriptions/ISic030033.xml

```diff
@@ -0,0 +1,208 @@
+                        <objectDesc>
+                            <supportDesc>
+                                <support>
+                                    <p>Lamina.</p>
+                                    <material ana="#material.metal.lead" ref="https://www.eagle-network.eu/voc/material/lod/108.html">lead</material>
+                                    <objectType ana="#object.lamina" ref="https://www.eagle-network.eu/voc/objtyp/lod/145.html">lamina</objectType>
+                                    <dimensions>
+                                        <height unit="cm"/>
+                                        <width unit="cm"/>
+                                        <depth unit="cm"/>
+                                    </dimensions>
+                                </support>
+                                <condition/>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout>
+                                    <rs ana="#execution.engraved" ref="https://www.eagle-network.eu/voc/writing/lod/167.html">engraved</rs>
+                                    <p></p>
+                                    <damage/>
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc>
+                        <handDesc>
+                            <handNote>
+                                <!-- Based on Glyph table: -->
+                                <locus from="line1" to="line1">Line 1</locus>
+                                <dimensions type="letterHeight">
+                                    <height unit="mm"/>
+                                </dimensions>
+                            </handNote>                       
+                        </handDesc>
+                    </physDesc>
+                    <history>
+                        <origin>
+                            <origPlace>
+                                <placeName/>
+                                <placeName type="ancient" ref="https://pleiades.stoa.org/places/462126">Camarina</placeName>
+                                <placeName type="modern">Camarina</placeName>
+                                <geo></geo> <!-- add geo coordinates -->
+                            </origPlace>
+                            <origDate datingMethod="#julian" notBefore-custom="-0460" notAfter-custom="-0450">460-450 BCE (so Arena on letters).</origDate>
+                        </origin>
+                        <provenance type="found">Found in the necropolis of Passo Marinaro (Camarina)<geo></geo>.</provenance> <!-- add geo coordinates -->
+                        <provenance type="observed" subtype="autopsied">No Autopsy</provenance>
+                        <acquisition></acquisition> <!-- add -->
+                    </history>
+                </msDesc>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
```

Comment:
this encoding desc (haven't checked, but so all the defixio template files here?) is also lacking the chardesc.xml file inclusion